src/pages/StoryForm/StoryForm1.css

```
@@ -153,6 +159,15 @@
   font-weight: 500;
 }
 
+.story-form-hint {
+  margin: 6px 0 0;
+
+  color: var(--color-text-muted);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)